<launch>
  
  <!-- joynode -->
  <node pkg="joy" 
        exec="joy_node"
        name="joy_node"
        output="screen">
    <param name="dev" value="/dev/input/js0"/>
    <param name="deadzone" value="0.05"/>

  <!-- joy manager node-->
  <node pkg="joy_manager"
        exec="joy_manager_node"
        name="joy_manager_node"
        output="screen">
        
    <remap from="/joy" to="/joy"/>
    <remap from="/ackermann_cmd" to="/ackermann_cmd"/>
    <remap from="/cmd_drive" to="/cmd_drive"/>

  </node>

  
</launch>